app/views/layouts/application.html.erb

```diff
@@ -13,6 +13,7 @@
 	<div class="container">
 		<%= yield %>
 		<%= render 'layouts/footer' %>
+		<%=debug(params) if Rails.env.development? %>
 	</div>
 </body>
 </html>
```

Comment:
Anh sử dụng setting này dành cho sublime để xử lý indent, trailing space và end of file nhé 
Preference -> Setting User -> xóa hết cái cũ và paste cái mới vào nhé
[code]
{
	"ensure_newline_at_eof_on_save": true,
	"font_size": 11,
	"highlight_line": true,
	"ignored_packages":
	[
		"Vintage"
	],
	"rulers":
	[
		80
	],
	"show_encoding": true,
	"show_line_endings": true,
	"tab_size": 2,
	"translate_tabs_to_spaces": true,
	"trim_trailing_white_space_on_save": true
}
[/code]